plugin/build.sbt

```diff
@@ -0,0 +1,10 @@
+name := "plugin"
+
+version := "0.1.1-SNAPSHOT"
+
+scalaVersion := "2.12.8"
+
+libraryDependencies ++= Seq(
+  "org.scala-lang" % "scala-compiler" % "2.12.8" withSources()
+)
+
```

Comment:
Why `2.12.8`, the rest of this repo is on `2.12.11`? Also do you know about the publishing rules for plugins? I believe they are binary incompatible at a Scala minor version level, ie. I think we need to publish this for `2.12.0-11`.

Reply:
Yup, I think i fixed it now.